tests/storage/datavolume.go

```diff
@@ -102,12 +102,12 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 
 				// This will only work on storage with binding mode WaitForFirstConsumer,
 				if tests.HasBindingModeWaitForFirstConsumer() {
-					tests.WaitForDataVolumePhaseWFFC(dataVolume.Namespace, dataVolume.Name, 30)
+					Eventually(ThisDV(dataVolume), 30).Should(BeInPhase(cdiv1.WaitForFirstConsumer))
 				}
 				num := 2
 				By("Starting and stopping the VirtualMachineInstance a number of times")
 				for i := 1; i <= num; i++ {
-					tests.WaitForDataVolumeReadyToStartVMI(vmi, 140)
+					Eventually(ThisDV(dataVolume), 180).Should(HaveSucceeded())
```

Comment:
This test is a little bit tricky since the body tries to cove two cases. I think writing this would be equivalent to the function which was called before:

```
Eventually(ThisDV(dataVolume), 180).Should(Or(HaveSucceeded(), BeInPhase(cdiv1.WaitForFirstConsumer)))
```

If this is needed too often, we could consider creating a helper matcher `BeReadyToBeConsumed` or something.